templates/registration/login.html

```diff
@@ -1,79 +1,95 @@
 {% extends "base.html" %}
 
 {% block media %}
-    <style>
-        h4 {
-            padding-top: 1em;
-        }
+<style>
+    h4 {
+        padding-top: 1em;
+    }
 
-        .social {
-            display: inline;
-            font-size: 2.3em;
-            float: none;
-        }
+    .social {
+        display: inline;
+        font-size: 2.3em;
+        float: none;
+    }
 
-        .google-icon i {
-            color: #DD4B38;
-        }
+    .google-icon i {
+        color: #DD4B38;
+    }
 
-        .facebook-icon i {
-            color: #133783;
-        }
+    .facebook-icon i {
+        color: #133783;
+    }
 
-        .github-icon i {
-            color: black;
-        }
-    </style>
+    .github-icon i {
+        color: black;
+    }
+</style>
 {% endblock %}
 
 {% block body %}
-    <div class="auth-flow-form">
-        <form action="" method="post" class="form-area">
-            {% csrf_token %}
-            {% if form.errors %}
-                <div id="form-errors">
-                    <p class="error">{{ _('Invalid username or password.') }}</p>
-                </div>
-            {% endif %}
-            <table border="0" style="text-align:left">
-                <tr>
-                    <th><i class="fa fa-user fa-fw"></i>
-                    </th>
-                    <td>{{ form.username }}
-                    </td>
-                </tr>
-                <tr>
-                    <th><i class="fa fa-key fa-fw"></i>
-                    </th>
-                    <td>{{ form.password }}
-                    </td>
-                </tr>
-            </table>
-            <hr>
-            <button style="float:right;" type="submit">{{ _('Login!') }}</button>
-            <input type="hidden" name="next" value="{{ next }}">
-        </form>
-        <a href="{{ url('password_reset') }}">{{ _('Forgot your password?') }}</a>
+<div class="auth-flow-form">
+    <form action="" method="post" class="form-area">
+        {% csrf_token %}
+        {% if form.errors %}
+        <div id="form-errors">
+            <p class="error">{{ _('Invalid username or password.') }}</p>
+        </div>
+        {% endif %}
+        <table border="0" style="text-align:left">
+            <tr>
+                <th><i class="fa fa-user fa-fw"></i>
+                </th>
+                <td>{{ form.username }}
+                </td>
+            </tr>
+            <tr>
+                <th><i class="fa fa-key fa-fw"></i>
+                </th>
+                <td>{{ form.password }}
+                </td>
+                <td><i class="fa fa-eye fa-lg">
```

Comment:
I think this lacks a bit when it comes to accessibility. We should do one of the following:

1. Use `aria-role="button"` alongside an appropriate `aria-label` (something like "Make password visible."). The `aria-pressed` attribute can be toggled too, [see the MDN docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role) and [the FontAwesome docs](https://fontawesome.com/v4.7/accessibility/).

2. Alternatively, just hiding with `aria-hidden="true"` may be just as acceptable because a screen reader could probably just read the password out loud.

Not sure what's best.

Reply:
@fluix-dev Thank you for reply even though it was a problematic code. After supplement the problems, conducting various tests, May I send the PR again?

Reply:
You can amend your previous commit with:
```
git commit --amend
```
and then force push it:
```
git push -f
```

Though no promises this gets accepted of course.